types/convert.go

```diff
@@ -292,7 +293,8 @@ func floatStrToIntStr(validFloat string) (string, error) {
 		extraZeroCount := intCnt - len(digits)
 		if extraZeroCount > 20 {
 			// Return overflow to avoid allocating too much memory.
```

Comment:
Should we update this comment?

Reply:
@CodeRushing Please update this comment.

Reply:
@zz-jason already updated